cmd/config-migrate.go

```diff
@@ -24,6 +24,9 @@ import (
 	"github.com/minio/minio/pkg/quick"
 )
 
+// DO NOT EDIT following message template, please open a github issue to discuss instead.
+var configMigrateMSGTemplate = "Minio configuration file %s migrated from version '%s' to '%s' successfully.\n"
```

Comment:
I think `Configuration file` is good enough than `Minio configuration file`.  As minio server shows this at startup `Minio` is redundant word.